examples/vertexai-run.yaml

```
@@ -0,0 +1,91 @@
+
+providers:
+  vector_io:
+    - config:
+        kvstore:
+          db_path: /tmp/faiss_store.db
+          type: sqlite
+      provider_id: faiss
+      provider_type: inline::faiss
+
+  agents:
+  - config:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)